build.gradle

```diff
@@ -54,7 +54,7 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
         languageVersion = '1.7'
         // Specify the Kotlin libraries that code is compatible with
         apiVersion = '1.7'
-        // Note that we Need to use a version of Kotlin that will be compatible with the Corda API.
+        // Use a version of Kotlin that will be compatible with the Corda API.

```

Comment:
From:

Use a version of Kotlin that will be compatible with the Corda API.

To:

Use a version of Kotlin that is compatible with the Corda API.